### DISCUSSION TITLE
Browsers can not open or save files

### DISCUSSION THREAD

**Discussion by davidystephenson** [regolith-linux/regolith-desktop#984] [Q&A] [answered]
After upgrading Regolith, I can no longer save or open files using Chrome or Firefox. I can still drag and drop files into website dropzones, but when I try to save with the context menu or upload with a file input, nothing happens. I've seen recommendations to install `xdg-desktop-portal-gnome`, but when I started apt warned me `regolith-desktop` and `regolith-system-ubuntu` would be removed. How can I re-enable file upload and saving?

**Comment by kgilmer** [comment, marked as answer]
Another user reported this behavior as well.  Until we implement this functionality in the Regolith portal implementation, your best path to resolving this is to uninstall both `xdg-desktop-portal-regolith` and `xdg-desktop-portal-gnome`.  If you're using the latest packages for `mantic`, you should be able to safely uninstall these portal packages without removing the rest of `regolith-desktop`.

**Reply by PatriceGendreau** [reply]
I had the problem.... but uninstall both files cures the problem
(sorry I'm french)

**Reply by davidystephenson** [reply]
Thank you, this fixed the problem.

**Reply by dariooddenino** [reply]
Thanks, this fixed the problem for me as well.

**Reply by cekingx** [reply]
thankyou

**Comment by dphov** [comment]
This is how I resolved this issue

In your `/usr/share/xdg-desktop-portal/portals/gtk.portal`
Edit the part
```
UseIn=gnome
```
to 
```
UseIn=gnome;i3;regolith
```
Restart the system
Voilà


Question: Maybe this change should be a part of installation/upgrade?
@kgilmer

**Reply by kenjiknk** [reply]
thanks man, saved my life! 
btw u still using that workaround, or they already solved this issue?

**Reply by dphov** [reply]
@Kenji1655 you are welcome ! I didn't follow up, can't say

**Reply by kgilmer** [reply]
@dphov thank you for the suggestion.  Regolith is provided entirely as packages.  As such, we cannot modify the contents of another package from one of ours due to policy.
